src/main.rs

```diff
@@ -309,6 +309,7 @@ async fn provide_interactive(
                     paths.push(el.path().into());
                 }
             }
+            paths.sort();
```

Comment:
Is this still needed?  It's a little unfortunate to have to do this, would be nice if it could be avoided.

Reply:
I debated with myself, glad you brought it up. Is it okay that the same content on two different machines (folders structured the same way) will create different hashes?

Reply:
hmm, seems to be we should sort these, as adding the same files on two machines should result in the same hash

Reply:
Hum, we need a way of hashing that works natively on sets: no matter which order we add files we'd like the same hash.  :smile:  not that that would help since we hash the metadata.